mllib/src/main/scala/org/apache/spark/mllib/feature/ChiSqSelector.scala

```diff
@@ -57,22 +69,21 @@ class ChiSqSelectorModel @Since("1.3.0") (
    * Preserves the order of filtered features the same as their indices are stored.
    * Might be moved to Vector as .slice
    * @param features vector
-   * @param filterIndices indices of features to filter
+   * @param filterIndices indices of features to filter, must be ordered asc
    */
   private def compress(features: Vector, filterIndices: Array[Int]): Vector = {
-    val orderedIndices = filterIndices.sorted
```

Comment:
You can refer https://github.com/apache/spark/pull/14597/files#diff-2accdf5bbf6d939eebe7727736fe6fdcR820. It breaks the binary compatibility and @mpjlu added it to ```MimaExcludes``` in that PR to make Jenkins happy. I think it's better to not break binary compatibility if there is no strong requirement. Thanks.